frontend/src/components/pages/Dashboard/components/DropoffCard.tsx

```diff
@@ -0,0 +1,116 @@
+import { StarIcon, TimeIcon } from "@chakra-ui/icons";
+import {
+  Badge,
+  Box,
+  HStack,
+  IconButton,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  Text,
+  useDisclosure,
+} from "@chakra-ui/react";
+import React from "react";
+import { useHistory } from "react-router-dom";
+
+import * as Routes from "../../../../constants/Routes";
+import { EllipsisIcon } from "../../../common/icons";
+import { SchedulingFormProps } from "../../Scheduling/types";
+import DeleteScheduleModal from "./DeleteScheduleModal";
+
+interface DropoffCardProps {
+  schedule: SchedulingFormProps;
+  onDelete: () => void;
+}
+
+const DropoffCard = ({ schedule, onDelete }: DropoffCardProps): JSX.Element => {
+  const startDate = new Date(schedule.startTime).toDateString();
+  const startTime = new Date(schedule.startTime).toLocaleTimeString();
+  const history = useHistory();
+
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  return (
+    <Box
+      maxW="sm"
+      mb="24px"
+      border="1px solid"
+      borderColor="hubbard.100"
+      borderRadius="8px"
+    >
+      <Box pl="6" pr="6" pb="6" pt="4">
+        <HStack spacing="10">
+          <Text mb="16px" textStyle="mobileBodyBold">
+            {startDate}
+          </Text>
+          <Menu isLazy>
+            <MenuButton
+              style={{
+                marginLeft: "6rem",
+                marginBottom: "15px",
+                marginRight: "0px",
+              }}
+              as={IconButton}
+              aria-label="options"
+              icon={<EllipsisIcon />}
+              variant="ghost"
+            />
+            <MenuList p={0} minW="0" w="94px">
+              <MenuItem
+                onClick={() =>
+                  history.push(`/${Routes.DASHBOARD_PAGE}/${schedule.id}`)
```

Comment:
I think u need to remove the extra `/` so like this: `${Routes.DASHBOARD_PAGE}/${schedule.id}`. I get this error if we have two `/`, since `DASHBOARD_PAGE` already has one. 
![image](https://user-images.githubusercontent.com/37384882/142171750-d892cd58-7fe2-4778-aca6-347147c69e66.png)